### ISSUE TITLE
Why is this doesn't work ?

### ISSUE THREAD

**Issue by TheIntelloBox** [DeadmanDungeons/AudioConnect#2]
Hi my friend ! :D 

I'm doing the command : /ac verify speedcraft.omgcraft.fr:12646 on my server and this say : The provided address did not verify as the public address for this server. 

Why ? :D 
Could you help me ? :D 
Thanks ! :D 
Have a nice morning/**day**/evening/night ! :D 
TheIntelloBox. :D

**Comment by JonMcPherson** [comment]
I have manually set your server address and am looking into the underlying issue,

**Comment by JonMcPherson** [comment]
I think I have an idea as to why it wasn't working for you. I noticed your server description (MOTD) is heavily customized and it looks like it is being modified by a plugin on the ServerListPingEvent.  The AudioConnect plugin listens to the ServerListPingEvent to modify the MOTD with an invisible verification code, and one of your plugins may be overwriting it.  So I changed the event listener in AudioConnect to the have the highest priority which should hopefully be enough prevent this.  This solution would only break if your plugin also has the highest priority and is enabled after AudioConnect on startup.

The plugin resource has been updated with this fix.
Commit: https://github.com/DeadmanDungeons/AudioConnect/commit/21582e6e57311469475092aee720c6b16936c406
